src/amo/components/Routes/index.js

```diff
@@ -25,9 +25,14 @@ import UserProfileEdit from 'amo/components/UserProfileEdit';
 import SimulateAsyncError from 'core/components/error-simulation/SimulateAsyncError';
 import SimulateClientError from 'core/components/error-simulation/SimulateClientError';
 import SimulateSyncError from 'core/components/error-simulation/SimulateSyncError';
+import type { ConfigType } from 'core/types/config';
+
+type Props = {|
```

Comment:
hmm not sure why this looks cut off to me...? It looks okay in my editor..

Reply:
Probably an issue with the "Refined GitHub" extension due to some GitHub changes (seeing the same).

Reply:
ahh ok...